### ISSUE TITLE
Installation Error.

### ISSUE THREAD

**Issue by YvanYin** [StanfordVL/MinkowskiEngine#123]
I try to install the lib on our uni cluster.
I used the coda to create the env and have installed the required packages.
The basic packages are : pytorch 1.3.0, torchvision 0.4.0, openblas 0.3.6.
The cuda version is 10.0.

When I use:
python setup.py install --blas=openblas --force
I get the errors as follows.

Using BLAS=openblas                                                                                                                                                                     Traceback (most recent call last):                                                                                                                                                        File "<string>", line 1, in <module>                                                                                                                                                  ModuleNotFoundError: No module named 'torch'                                                                                                                                            Traceback (most recent call last):                                                                                                                                                        File "<string>", line 1, in <module>                                                                                                                                                  ModuleNotFoundError: No module named 'torch'                                                                                                                                            Traceback (most recent call last):                                                                                                                                                        File "<string>", line 1, in <module>                                                                                                                                                  ModuleNotFoundError: No module named 'torch'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'torch'           

It said that 'torch' is not found. 
However, I have successfully installed pytroch-1.3.0.
        
Can you help me to fix this?

**Comment by chrischoy** [comment]
If it could not import torch, it might be installed in a different environment or python and pip are using different versions.

Try

```
python -c "import torch; print(torch.__version__)"
```

**Comment by YvanYin** [comment]
Hi,
I have tried 
`python -c "import torch; print(torch.__version__)"`
It can output the version of pytorch, '1.3.0'.

I found that when the code run to the line #118 
'if not KEEP_OBJS:
    run_command('make', 'clean')'
It will output error.
'
Using BLAS=openblas 
Traceback (most recent call last): 
      File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'torch' 
'
I also directly run `make clean`, it will output the same error. It seems that I should modify the Makefile  to make it find the right python path.

Besides, before I run the `python setup.py install --blas=openblas --force`, I should link the python to the directory of my environment:
`alias=~/.conda/envs/pytorch-130/bin/python`
otherwise, it will cannot find this python but another one.
